charts/hami/values.yaml

```diff
@@ -85,6 +85,9 @@ scheduler:
     host: 127.0.0.1 # hostname or ip, can be your node'IP if you want to use https://<nodeIP>:<schedulerPort>/<path>
     port: 31998
     path: /webhook
+    whitelistNamespaces:
```

Comment:
I recommend leaving the default value empty. This is just an entry point for end users to customize.

Reply:
Done, updated it with a comment using the template.